### PR TITLE
style(ui): Move "add remotes" button out of peer dropdown

### LINKED ISSUE #1866: Move "Manage Remotes" into it's own button?
The button to collaborate with other users is hidden under a dropdown. It should be move obvious where this is and that this button allows you to collaborate with other Radicle users.

Maybe the button could simply be moved next to the dropdown.

image

### PR DESCRIPTION
**Pull request by juliendonck**
closes #1866

### CODE REVIEW

File: ui/DesignSystem/Component/FollowToggle.svelte [hunk 1]
```diff
@@ -48,5 +48,7 @@
   on:click={click}
   on:mouseenter={mouseenter}
   on:mouseleave={mouseleave}>
-  {title}
+  {#if !following || (following && hovering) || (following && disabled && hovering)}
```
Comment (geigerzaehler) [inline ui/DesignSystem/Component/FollowToggle.svelte:51]: This is the same as
```suggestion
  {#if !following || hovering}
```

File: ui/DesignSystem/Primitive/Button.svelte [hunk 2]
```diff
@@ -320,9 +320,7 @@
   on:click
   on:mouseenter
   on:mouseleave
-  {style}>
+  style={`${style}` + "gap: .5rem;"}>
```
Comment (geigerzaehler) [inline ui/DesignSystem/Primitive/Button.svelte:323]: Shouldn't the `gap` property be added to the `button` class?

File: cypress/integration/networking.spec.ts [hunk 3]
```diff
@@ -124,12 +124,6 @@ context("p2p networking", () => {
           commands
             .pickWithContent(["followed-peers", "peer-abbey"], shortenedPeerId)
             .should("exist");
-          commands
-            .pickWithContent(
-              ["followed-peers", "peer-abbey", "follow-toggle"],
-              "Following"
-            )
-            .should("exist");
```
Comment (geigerzaehler) [inline cypress/integration/networking.spec.ts:132]: We can ignore the content but we should still check the toggle.
```ts
commands.pick("followed-peers", "peer-abbey", "follow-toggle").should("exist")
```

File: ui/DesignSystem/Component/FollowToggle.svelte (hunk 1 above)
Comment (juliendonck) [reply ui/DesignSystem/Component/FollowToggle.svelte:51]: lol

File: cypress/integration/networking.spec.ts (hunk 3 above)
Comment (juliendonck) [reply cypress/integration/networking.spec.ts]: https://github.com/radicle-dev/radicle-upstream/pull/1928/commits/6e8a1b09c376d1fb368842365054e938de5aefe7

File: ui/DesignSystem/Primitive/Button.svelte (hunk 2 above)
Comment (juliendonck) [reply ui/DesignSystem/Primitive/Button.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/1928/commits/6e8a1b09c376d1fb368842365054e938de5aefe7

**Review by geigerzaehler — APPROVED** [review]

**Review by geigerzaehler — APPROVED** [review]

**Pull request merged**